src/mongocxx/private/libmongoc_symbols.hh

```diff
@@ -154,6 +154,7 @@ MONGOCXX_LIBMONGOC_SYMBOL(client_pool_push)
 MONGOCXX_LIBMONGOC_SYMBOL(client_pool_set_apm_callbacks)
 MONGOCXX_LIBMONGOC_SYMBOL(client_pool_try_pop)
 MONGOCXX_LIBMONGOC_SYMBOL(client_reset)
+MONGOCXX_LIBMONGOC_SYMBOL(client_select_server);
```

Comment:
The trailing ; is not needed, and I think the cause of some compilation warnings on Evergreen.

Reply:
Done